src/app/Scenes/Activity/ActivityContainer.tsx

```diff
@@ -0,0 +1,49 @@
+import { ActivityContainerQuery } from "__generated__/ActivityContainerQuery.graphql"
+import { useMarkNotificationsAsSeen } from "app/Scenes/Activity/hooks/useMarkNotificationsAsSeen"
+import { Suspense } from "react"
+import { graphql, useLazyLoadQuery } from "react-relay"
+import { ActivityList } from "./ActivityList"
+import { ActivityTabPlaceholder } from "./ActivityTabPlaceholder"
+import { NotificationType } from "./types"
+import { getNotificationTypes } from "./utils/getNotificationTypes"
+
+interface ActivityProps {
+  type: NotificationType
+}
+
+export const ActivityContainer: React.FC<ActivityProps> = (props) => {
+  return (
+    <Suspense fallback={<ActivityTabPlaceholder />}>
+      <ActivityContent {...props} />
+    </Suspense>
+  )
+}
+
+export const ActivityContent: React.FC<ActivityProps> = ({ type }) => {
```

Comment:
This export might not be needed, but also ok to leave it.
```suggestion
const ActivityContent: React.FC<ActivityProps> = ({ type }) => {
```